osxcollector/output_filters/summary_filters/html.py

```diff
@@ -327,4 +329,11 @@ def _print_val(self, val):
                 self._write('</li>')
             self._write('</ul>')
         elif isinstance(val, basestring) or isinstance(val, Number):
+            self._encode_val(val)
+
+    def _encode_val(self, val):
+        try:
             self._write(u'<span class="val">{0}</span>'.format(val))
+        except UnicodeDecodeError as err:
+            sys.stderr.write('Unicode decode error while processing the value: {0}\n{1}\n'.format(val, err))
+            self._write('<span class="val">{0}</span>'.format(val))
```

Comment:
Right, so it seems these cases are then caught in the `except` statement in `_write()` method.

What is weird is that the characters are printed correctly in the console, but not in the HTML file, as the browser is trying to decipher them as UTF-8.